src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  position: relative;
+}
+
+.nav_item {
+  display: inline-block;
+  margin-left: 20px;
+}
+
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-transform: uppercase;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: Roboto;
```

Comment:
You should use fallback fonts - alternative font-family in case the main one doesn't work. This helps to ensure that your website is displayed correctly on different devices and browsers. For example, you can modify your code as follows: `font-family: Roboto, Arial, sans-serif;`